scripts/ci-tool.py

```diff
@@ -165,6 +165,10 @@ def parse_commandline():
                         help='Run the etch language tests')
     parser.add_argument('--lint', action='store_true',
                         help='Run clang-tidy')
+    parser.add_argument('--fix-lint', action='store_true',
+                        help='Run clang-tidy and attempt to fix lint errors')
+    parser.add_argument('-C', '--commit', nargs=1, default=None,
```

Comment:
Use lowercase `-c` for consistency with `apply_style.py`

Reply:
Are you sure? All other flags are uppercase for this script